hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/FinalizeChildRecordHandler.java

```diff
@@ -72,6 +72,9 @@ public void finalizeChildRecord(@NonNull final ChildFinalizeContext context, fin
             // set an empty transfer list even if there are no hbar changes but the parent record succeeded
             // to be compatible with mono-service
             recordBuilder.transferList(TransferList.DEFAULT);
+        } else if (recordBuilder.status() != ResponseCodeEnum.SUCCESS) {
```

Comment:
Is this still required after merging `develop`? C.f. [this change](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java#L629) which initializes child `transferList` to `null`.

Reply:
Even with the recent changes the `transferList` is still present in the result.